arm-recoveryservicesbackup/readme.md

```diff
@@ -31,20 +31,24 @@ directive:
 
 ### C#
 
-```yaml
```

Comment:
Please tell me what can I do about this? $git checkout upstream/master -- readme.md wont really change anything since the change is already done. It is just showing the diff here with the older file. Any way I can change that?